tests/texture.py

```diff
@@ -338,6 +338,67 @@ def texture_3d(cfg: SceneCfg):
     return render
 
 
+_STEPS = 4
+
+_RENDER_TO_TEXTURE3D_VERT = """
+void main()
+{
+    ngl_out_pos = ngl_projection_matrix * ngl_modelview_matrix * vec4(ngl_position, 1.0);
+}
+"""
+
+_RENDER_TO_TEXTURE3D_FRAG = f"""
+void main()
+{{
+    float steps = {_STEPS}.0;
```

Comment:
do you mind passing down an uniform instead of doing some templating?

Reply:
Done.